### DISCUSSION TITLE
3d model import android

### DISCUSSION THREAD

**Discussion by saggeldi** [maplibre/maplibre-native#2124] [Q&A]
I have android jetpack compose app build with own map tile server, using maplibre android library. I want add 3d model with textures from file obj, gltf,...etc. Like yandex maps. How Can I do that?

**Comment by saggeldi** [comment]
![Screenshot from 2024-02-20 15-16-41](https://github.com/maplibre/maplibre-native/assets/64743718/41dc95e8-3d09-4891-b5aa-91b44f939646)
Like this

**Reply by hmtriit** [reply]
Hi @shageldi-dev 
I also have the same question as you. Currently, I don't know if you have any ideas for implementing or further developing it with maplibre-native. Could you share any thoughts?

Thanks!

**Reply by bugraongun** [reply]
Have you found a solution?

**Reply by hmtriit** [reply]
> Have you found a solution?

Currently, I have tried to explore the tiny_objloader library to load models, but due to my limited experience with C/C++, the implementation has been quite difficult and remains incomplete.
